index.html

```diff
@@ -29,352 +32,374 @@
+                  <li><img src="images/c1.png" alt="" /></li>
+                  <li><a href="" class="text-3">2015</a></li>
+                </ul>
+              </div>
+              <div>
+                <p>
+                  Experimental content creation feature that allows users to add
+                  to an existing story over the course of a day without spamming
+                  their friends.
+                </p>
+              </div>
+              <div class="base-links">
+                <ul>
+                  <li><a href="" class="base-link-text1">html</a></li>
+                  <li><a href="" class="base-link-text2">Ruby.on.rails</a></li>
+                  <li><a href="" class="base-link-text2">css</a></li>
+                  <li><a href="" class="base-link-text3">javascript</a></li>
+                </ul>
+              </div>
+              <div class="base-button">
+                <a href="#">See Project</a>
+              </div>
+            </div>
+            <div class="img-container2-dt">
+              <img src="images/desktop-section-2.png" alt="" />
             </div>
-          <div class = "base-links">
-            <ul>
-              <li><a href="" class="base-link-text1">html</a></li>
-              <li><a href = "" class = "base-link-text2">Ruby.on.rails</a></li>
-              <li><a href="" class="base-link-text2">css</a></li>
-              <li><a href="" class="base-link-text3">javascript</a></li>
-            </ul>
-          </div>
-          <div class = "base-button">
-          <a href = "#">See Project</a>
-          </div>
           </div>
-  
-  </div>
-        
-        <!----End Markup for Desktop styling 3-->
-        <div class="s3-container">
-          <div class="image-container3"></div>
+          <!---End Mark up for s2 Desktop -->
 
-          <h1>Tonic</h1>
-          <div class="list-container">
-            <ul>
-              <li><a href="" class="text">CANOPY</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-2">Back End Dev</a></li>
-              <li><img src="images/c1.png" alt="image of a dot" /></li>
-              <li><a href="" class="text-3">2015</a></li>
-            </ul>
-          </div>
-          <div>
-          <p>
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
-          </p>
+          <!------End of styling for Desktop card 2-->
+          <div class="s2-container">
+            <div class="image-container2"></div>
+
+            <h2 class="newh2">Multi-Post-Stories</h2>
+            <div class="list-container">
+              <ul>
+                <li><a href="" class="text">CANOPY</a></li>
+                <li><img src="images/c1.png" alt="" /></li>
+                <li><a href="" class="text-2">Back End Dev</a></li>
+                <li><img src="images/c1.png" alt="" /></li>
```

Comment:
- [ ]  I noticed here the alt is empty,, kindly add a text describing the image for accessibility issues 🙏🏻